### PR TITLE
Add support for client bounds without NonClient Area on Windows

### PR DESCRIPTION
**Pull request by Pavou**
Add support for `activeWindow()` to return `clientBounds` which is the bounds of the client area of the window. 

The client area of the window does not include Nonclient Area such as the title bar, menu bar, or window frame.  

This is useful if the client position and size is used to add overlay windows and dialogs on top of the activeWindow. 

The change is only for Windows platform and only the relevant typescript definitions and tests were updated.

### CODE REVIEW

File: Sources/windows/main.cc [hunk 1]
```diff
@@ -194,12 +197,26 @@ Napi::Value getWindowInformation(const HWND &hwnd, const Napi::CallbackInfo &inf
 	owner.Set(Napi::String::New(env, "path"), ownerInfo.path);
 	owner.Set(Napi::String::New(env, "name"), ownerInfo.name);
 
+  // bounds window
```
Comment (sindresorhus) [inline Sources/windows/main.cc:200]: Incorrect indentation.

File: index.d.ts [hunk 2]
```diff
@@ -95,6 +95,16 @@ export type LinuxResult = {
 
 export type WindowsResult = {
 	platform: 'windows';
+
+	/**
+		 Client position and size.
+	 */
+	boundsClient: {
```
Comment (sindresorhus) [inline index.d.ts:102]: `contentBounds` would be a better name.

File: index.d.ts [hunk 3]
```diff
@@ -95,6 +95,16 @@ export type LinuxResult = {
 
 export type WindowsResult = {
 	platform: 'windows';
+
+	/**
+		 Client position and size.
```
Comment (sindresorhus) [inline index.d.ts:100]: Indentation

Comment (sindresorhus) [inline index.d.ts:100]: Needs a better description of what it is.

File: readme.md [hunk 4]
```diff
@@ -80,6 +80,11 @@ Returns a `Promise<object>` with the result, or `Promise<undefined>` if there is
 	- `y` *(number)*
 	- `width` *(number)*
 	- `height` *(number)*
+- `boundsClient` *(Object)* - Client position and size without Nonclient Area *(Windows only)*
```
Comment (sindresorhus) [inline readme.md:83]: The text here should be in sync with index.d.ts

**Review by sindresorhus — CHANGES_REQUESTED** [review]

**Comment by Pavou** [comment]
Thanks for the feedback. Updated.

File: index.d.ts [hunk 5]
```diff
@@ -95,6 +95,16 @@ export type LinuxResult = {
 
 export type WindowsResult = {
 	platform: 'windows';
+
+	/**
+	Content position and size without Nonclient Area
```
Comment (sindresorhus) [inline index.d.ts:100]: It's not obvious what "non client area" means. Spell it out for the user.

Comment (Pavou) [reply index.d.ts:100]: `Content position and size without Nonclient Area i.e. without the title bar, menu bar, and window frame`

Is this ok?

Comment (sindresorhus) [reply index.d.ts:100]: How about:

> Window content position and size, which excludes the title bar, menu bar, and frame.

Comment (Pavou) [reply index.d.ts]: Sounds great. pushed.

**Pull request merged**